app/services/account_reset/delete_account.rb

```diff
@@ -66,13 +66,20 @@ def notify_user_via_email_of_deletion
     end
     # rubocop:enable IdentityIdp/MailLaterLinter
 
+    def profile_components
+      return nil if !user.identity_verified?
+      user.active_profile.proofing_components if user.active_profile?
```

Comment:
The `user.active_profile?` check does what we wanted with `identity_verified?`, so we don't need that anymore.

https://github.com/18F/identity-idp/blob/e920584bfc4e8a1004763b6297504a2c9fd9a76f/app/models/user.rb#L375-L377

```suggestion
      user.active_profile.proofing_components if user.active_profile?
```

Reply:
I am getting an error for Anonymous users (active_profile? missing method) I'm trying to do as a 1 liner now with [e07200c](https://github.com/18F/identity-idp/pull/11236/commits/e07200c0c1520563a54b411f7763111478e5d27f)

Reply:
I think we should add the stubbed `active_profile?` method to `AnonymousUser` instead.

Reply:
That makes sense. I have added that to the AnonymousUser class.